test/media/vtt_text_parser_unit.js

```diff
@@ -368,7 +368,8 @@ describe('VttTextParser', function() {
   function verifyHelper(cues, text, opt_offset) {
     var data = shaka.util.StringUtils.toUTF8(text);
     // Last two parameters are only used by mp4 vtt parser.
-    var result = shaka.media.VttTextParser(data, opt_offset || 0, null, null);
+    var result =
+        shaka.media.VttTextParser(data, opt_offset || 0, null, null, null);
```

Comment:
Let's also add a new test "uses relative time stamps if configured to" which will pass "true" to the vtt parser and check that the start/end time of the cues is now relative.